docs/configuration/index.md

```diff
@@ -353,6 +353,7 @@ The following monitors are available:
 |`org.apache.druid.segment.realtime.RealtimeMetricsMonitor`|Reports statistics on Realtime processes.|
 |`org.apache.druid.server.metrics.EventReceiverFirehoseMonitor`|Reports how many events have been queued in the EventReceiverFirehose.|
 |`org.apache.druid.server.metrics.QueryCountStatsMonitor`|Reports how many queries have been successful/failed/interrupted.|
+|`org.apache.druid.server.metrics.TaskCountStatsMonitor`|Reports how many tasks are success/failed/running/pending/waiting.|
```

Comment:
As someone new to the monitors, I think explaining why you would need this is helpful. Are there specific use cases where I should be monitoring the task counts? Maybe I want to set up alerting on failed jobs?

Reply:
@suneet-s  Thank you very much for your reply! 

Some implementation classes are registered by default and we don't need to register them manually, such as **CacheMonitor**, **JettyMonitor** and **ExecutorServiceMonitor**. But some are not registered by default and need to be configured ourselves, such as **QueryCountStatsMonitor** and **TaskCountStatsMonitor**. 
Docs: https://druid.apache.org/docs/latest/operations/metrics.html
PR: https://github.com/apache/druid/pull/6657/files

I found a description for **QueryCountStatsMonitor**, but not for **TaskCountStatsMonitor**. The description of **TaskCountStatsMonitor** might have been missed, so I added.

In our scenario, kafka has a large number of indexing tasks and a large amount of data that needs to be consumed with low latency. Therefore, task monitoring is very important.

We deployed the druid cluster with ambari, so monitoring and alerting were implemented based on ambari-metric-collector.